src/module_8/AbstractDAOImpl.java

```diff
@@ -0,0 +1,53 @@
+package module_8;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class AbstractDAOImpl<User> implements AbstractDAO<User> {
+    ArrayList<User> dao = new ArrayList<User>();
+
+    @Override
+    public List save(List<User> list) {
+        dao.addAll(list);
+        return list;
+    }
+
+    @Override
+    public void delete(List<User> list) {
+        dao.remove(list);
+    }
+
+    @Override
+    public void deleteAll(List<User> list) {
+        dao.removeAll(list);
+    }
+
+    @Override
+    public void saveAll(List<User> list) {
+        dao.addAll(list);
+    }
+
+    @Override
+    public List getList() {
+        return dao;
+    }
+
+    @Override
+    public void deleteById(long id) {
+        for (int i = 0; i < dao.size(); i++)
```

Comment:
get() method would be quite useful here. Use get() instead of manually seeking user